docs/cloud-session-launch-flags.md

```
@@ -0,0 +1,443 @@
+
+### T6: `claude --cloud "<description>" --disallowedTools Monitor`（pty）
+exit=0 → 受理される。エラーなし、セッション作成
+```
+Created cloud session: ctw probe
+View: https://claude.ai/code/session_013r58uU5RNBJzqRHwLe5Z8v?from=cli&m=0
+Resume with: claude --teleport session_013r58uU5RNBJzqRHwLe5Z8v
+```
+PRD 4.2 が記載する `Error: a cloud session does not enforce tool restrictions yet` は 2.1.247 では再現しなかった。
+
+### T7: 複合受理プローブ（pty）
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)